packages/taikoon/script/js/4everland.js

```diff
@@ -0,0 +1,166 @@
+const { S3 } = require("@aws-sdk/client-s3");
+const { Upload } = require("@aws-sdk/lib-storage");
+const fs = require("fs");
+const fsPromises = fs.promises;
+const path = require("path");
+
+async function uploadFile(s3, params) {
+  try {
+    const task = new Upload({
+      client: s3,
+      queueSize: 3, // 3 MiB
+      params,
+    });
+
+    const res = await task.done();
+    return res.ETag.split('"').join("");
+  } catch (error) {
+    if (error) {
+      console.log("task", error.message);
+    }
+  }
+}
+
+// Helper function to form the metadata JSON object
+function populateNFTMetadata(name, description, CID) {
+  return {
+    name,
+    description,
+    image: CID,
+  };
+}
+
+async function main() {
+  const s3Params = {
+    accessKey: "KYEKUCIU01RUW4LL5M26",
```

Comment:
I don't think this should be public. If this is the actual credentials, please invalidate them